kasa/device.py

```diff
@@ -96,7 +101,9 @@ async def connect(
         """
         from .device_factory import connect  # pylint: disable=import-outside-toplevel
 
-        return await connect(host=host, config=config)  # type: ignore[arg-type]
+        # TODO: Incompatible return value type (got "Device[Any]", expected "T")
```

Comment:
This mypy error goes away with the change suggested above

Reply:
Removed.